src/Libraries/CoreNodes/Color.cs

```diff
@@ -260,7 +265,12 @@ public static Color Add(Color c1, Color c2)
                 c1.Green + c2.Green,
                 c1.Blue + c2.Blue);
         }
-
+        /// <summary>
+        /// Returns a new color after multiplying an input color's ARGB values with an input multiplier.
```

Comment:
Change to 'Multiply an input color with a number to produce a darker color. Input color must have an alpha less than 255.'